### DISCUSSION TITLE
[2022 HOMEWORK TURN-IN] Post links to "turn in" your homework here.

### DISCUSSION THREAD

**Discussion by williamgilpin** [williamgilpin/cphy#3] [archived]
Please post a link to your completed assignments in this thread. The schedule and syllabus are [here](https://docs.google.com/document/d/1URJmdpTVG8E2bLLu5xAHctICb6krbZ0fC0hO2i2xEXY/edit?usp=sharing)

Feel free to either: Link to the assignment in a fork of the full course repo; link to just the completed assignment (preferably still host it on your GitHub); or upload a .ipynb or .py file using [GitHub gists](https://gist.github.com/).

+ Since the course and assignments are brand new, in 2022 assignments are a completion grade, and solutions will be posted at least a week before the due date. Please feel free to adapt your solution from my code if you're stuck.

+ Please put your name in your post somewhere, so I know who's who

**Comment by adamscottc** [comment]
https://github.com/adamscottc/cphy/blob/main/HW1_Adam_Christensen.ipynb

**Reply by williamgilpin** [reply]
Adam Scott

**Comment by engineer9001** [comment]
https://github.com/engineer9001/HW1

**Reply by williamgilpin** [reply]
Dalton Myers

**Comment by qianggao-lab** [comment]
https://github.com/ustcer0928/cphyhw

**Reply by williamgilpin** [reply]
Qiang Gao

**Comment by swilliamson7** [comment]
https://github.com/swilliamson7/comp_physics_sw/blob/main/hw/hw1_solutions_sw.ipynb

**Comment by jiangzz-lab** [comment]
https://github.com/jiangzz-lab/cphy_hw/tree/main/hw1
Zhanzhi Jiang

**Comment by micontre** [comment]
[My course repo](https://github.com/micontre/computational_physics) (links to the hws in the README)
or if you prefer: 

- [Abelian Sandpile](https://github.com/micontre/computational_physics/blob/main/code/hw1/abelian_sandpile.ipynb)
- [Percolation](https://github.com/micontre/computational_physics/blob/main/code/hw1/percolation.ipynb)

Michelle Contreras Cossio :)

**Comment by liuyuezhang** [comment]
https://github.com/liuyuezhang/cphy/blob/main/hw/cellular_automata_complexity.ipynb

Liu, Yuezhang.

**Comment by jakemcgrath1999** [comment]
https://github.com/jakemcgrath1999/comp_phys

Jake McGrath's hw solutions

**Comment by maxwellsdemon1965** [comment]
https://github.com/maxwellsdemon1965/cphy-homework/blob/main/cellular_automata_complexity_hw.ipynb

Hector Iglesias

**Comment by DeepeshBVerma** [comment]
https://github.com/DeepeshBVerma/PHY381C.git

Deepesh Verma (dbv293)

**Comment by PatrickGio** [comment]
https://github.com/PatrickGio/Phy381_Computational_Physics/tree/main/Lab_1

Patrick Giolando (pag2636)

**Comment by iraklolo** [comment]
https://github.com/iraklolo/PHY381_HW1

Irakli Jokhadze

**Comment by domstec** [comment]
https://github.com/domstec/cphy/blob/main/CPHY_HW1_DSTEC.ipynb

Dominik Stec (dls4727)

**Comment by ihkim94** [comment]
https://github.com/ihkim94/cphy_WG/blob/main/cphy/hw/cellular_automata_complexity.ipynb

Inhwan Kim

**Comment by Bunch1RechargerUnknown** [comment]
https://github.com/crchristian/CompPhysHW/blob/main/HW1_caseychristian.ipynb

Casey Christian

**Comment by ramreddy-physics** [comment]
[HW1](https://github.com/ramreddy-physics/cellular_automata)

Ram Reddy Sudha

**Comment by erfanhamdi** [comment]
https://github.com/erfanhamdi/cphy_problems/blob/main/cellular_automata/README.md

just the Abelian sandpile

**Comment by Tariq-Mahassen** [comment]
https://github.com/Tariq-Mahassen/Tariq-Mahassen.git

**Comment by swilliamson7** [comment]
HW2 solutions are [here](https://github.com/swilliamson7/comp_physics_sw/blob/main/hw/hw2_solutions_sw.ipynb)

Sarah Williamson

**Comment by engineer9001** [comment]
HW2 solutions [here](https://github.com/engineer9001/HW2)

(Dalton Myers)

**Comment by jiangzz-lab** [comment]
https://github.com/jiangzz-lab/cphy_hw/tree/main/hw2

Zhanzhi Jiang

**Comment by adamscottc** [comment]
https://github.com/adamscottc/cphy/blob/main/HW2_Adam_Christensen.ipynb

Adam Christensen

**Comment by qianggao-lab** [comment]
https://github.com/ustcer0928/cphyhw/blob/master/Homework/matrices_unsupervised_learning.ipynb

Qiang Gao

**Comment by liuyuezhang** [comment]
https://github.com/liuyuezhang/cphy/blob/main/hw/matrices_unsupervised_learning-lyz.ipynb

Liu, Yuezhang

**Comment by maxwellsdemon1965** [comment]
Homework 2 solutions [here](https://github.com/maxwellsdemon1965/cphy-homework/blob/main/matrices_unsupervised_learning_hw.ipynb):

Hector Iglesias

**Comment by iraklolo** [comment]
https://github.com/iraklolo/PHY381_HW2

Irakli Jokhadze

**Comment by DeepeshBVerma** [comment]
https://github.com/DeepeshBVerma/PHY381C/blob/master/hw2.ipynb

Deepesh Verma (dbv293) let's gooooooo!

**Comment by jakemcgrath1999** [comment]
Jake's homework

https://github.com/jakemcgrath1999/comp_phys

**Reply by jakemcgrath1999** [reply]
Pt 1: https://github.com/jakemcgrath1999/comp_phys/blob/main/power_method.ipynb
Pt 2: https://github.com/jakemcgrath1999/comp_phys/blob/main/PCA.ipynb

**Comment by micontre** [comment]
[My course repo](https://github.com/micontre/computational_physics) 
[Power Method](https://github.com/micontre/computational_physics/blob/main/code/hw2/power_method.ipynb)
[PCA](https://github.com/micontre/computational_physics/blob/main/code/hw2/pca.ipynb)
Michelle Contreras Cossio

**Comment by Bunch1RechargerUnknown** [comment]
https://github.com/crchristian/CompPhysHW

Casey Christian

**Comment by ihkim94** [comment]
https://github.com/ihkim94/cphy_WG/blob/main/cphy/hw/matrices_unsupervised_learning.ipynb
Inhwan Kim

**Comment by ramreddy-physics** [comment]
[HW2](https://github.com/ramreddy-physics/Unsupervised-Learning)

Ram Reddy

**Comment by domstec** [comment]
https://github.com/domstec/cphy/blob/20137317316e491c384ef6cdb113849cbd735bda/CPHY_HW2_DSTEC.ipynb

Dominik Stec

**Comment by Tariq-Mahassen** [comment]
HW2
https://github.com/Tariq-Mahassen

**Comment by qianggao-lab** [comment]
HW #3
https://github.com/ustcer0928/cphyhw/blob/master/Homework/pde_turing.ipynb
Qiang Gao

**Comment by engineer9001** [comment]
HW3

https://github.com/engineer9001/HW3

Dalton Myers

**Comment by jiangzz-lab** [comment]
HW 3 https://github.com/jiangzz-lab/cphy_hw/tree/main/hw3
Zhanzhi Jiang

**Comment by DeepeshBVerma** [comment]
https://github.com/DeepeshBVerma/PHY381C/blob/master/hw3.ipynb

Deepesh Verma (dbv293)

**Comment by swilliamson7** [comment]
[HW 3 solutions! ](https://github.com/swilliamson7/comp_physics_sw/blob/main/hw/hw3_solutions_sw.ipynb)

Sarah Williamson

**Comment by jakemcgrath1999** [comment]
https://github.com/jakemcgrath1999/comp_phys/tree/main/hw3

Jake's hw 3 solutions

**Comment by iraklolo** [comment]
https://github.com/iraklolo/PHY381_HW3

Irakli Jokhadze

**Comment by maxwellsdemon1965** [comment]
Hector's homework 3 solutions [here](https://github.com/maxwellsdemon1965/cphy-homework/blob/main/pde_turing_hw.ipynb)

Hector Iglesias

**Comment by Bunch1RechargerUnknown** [comment]
https://github.com/crchristian/CompPhysHW/blob/main/HW3_caseychristian.ipynb
casey christian

**Comment by adamscottc** [comment]
https://github.com/adamscottc/cphy/blob/main/HW3_Adam_Christensen.ipynb
Adam Christensen

**Comment by ihkim94** [comment]
https://github.com/ihkim94/cphy_WG/blob/main/cphy/hw/pde_turing.ipynb
Inhwan Kim

**Comment by ramreddy-physics** [comment]
[HW3](https://github.com/ramreddy-physics/pde_PHY381C)

-Ram Reddy

**Comment by micontre** [comment]
[my repo](https://github.com/micontre/computational_physics)
[Allen-Cahn](https://github.com/micontre/computational_physics/blob/main/code/hw3/allen_cahn.ipynb)
[Turing](https://github.com/micontre/computational_physics/blob/main/code/hw3/turing.ipynb)
Michelle Contreras Cossio

**Comment by liuyuezhang** [comment]
[hw3](https://github.com/liuyuezhang/cphy/blob/main/hw/pde_turing-lyz.ipynb)

Liu, Yuezhang.

**Comment by domstec** [comment]
https://github.com/domstec/cphy/blob/main/CPHY_HW3_DSTEC.ipynb

Dominik Stec

**Comment by adamscottc** [comment]
https://github.com/adamscottc/cphy/blob/main/HW4_Adam_Christensen.ipynb

HW4

**Comment by engineer9001** [comment]
https://github.com/engineer9001/HW4

Dalton Myers' HW4

**Comment by qianggao-lab** [comment]
https://github.com/ustcer0928/cphyhw/blob/master/Homework/forecasting_regression_supervised.ipynb
Qiang Gao

**Comment by swilliamson7** [comment]
[HW4 solutions](https://github.com/swilliamson7/comp_physics_sw/blob/main/hw/hw4_solutions_sw.ipynb)

Sarah Williamson

**Comment by maxwellsdemon1965** [comment]
[HW4 submission](https://github.com/maxwellsdemon1965/cphy-homework/blob/main/forecasting_regression_supervised_hw.ipynb)

Hector Iglesias

**Comment by micontre** [comment]
[my repo](https://github.com/micontre/computational_physics)
[hw4](https://github.com/micontre/computational_physics/blob/main/code/hw4/ridge_regression.ipynb)

Michelle Contreras Cossio

**Comment by iraklolo** [comment]
https://github.com/iraklolo/iraklolo-PHY381_HW4

Irakli Jokhadze

**Comment by ihkim94** [comment]
https://github.com/ihkim94/cphy_2022/blob/main/HW_sub/forecasting_regression_supervised.ipynb

Inhwan Kim

**Comment by jiangzz-lab** [comment]
https://github.com/jiangzz-lab/cphy_hw/tree/main/hw4

hw4 Zhanzhi Jiang

**Comment by liuyuezhang** [comment]
https://github.com/liuyuezhang/cphy/blob/main/hw/forecasting_regression_supervised-lyz.ipynb

hw4 Liu, Yuezhang

**Comment by DeepeshBVerma** [comment]
https://github.com/DeepeshBVerma/PHY381C/blob/master/hw4.ipynb
https://github.com/DeepeshBVerma/PHY381C/blob/master/phy381_hw4.pdf
Deepesh Verma (dbv293)

**Comment by ramreddy-physics** [comment]
[HW4](https://github.com/ramreddy-physics/ridge_regression.git)

-Ram Reddy

**Comment by jakemcgrath1999** [comment]
https://github.com/jakemcgrath1999/comp_phys/tree/main/hw4

Jake

**Comment by domstec** [comment]
https://github.com/domstec/cphy/blob/main/CPHY_HW4_DSTEC.ipynb

Dominik Stec

**Comment by Bunch1RechargerUnknown** [comment]
https://github.com/crchristian/CompPhysHW/blob/main/HW4_caseychristian.ipynb

casey christian
